pkg/core/auth/session/redis/store.go

```diff
@@ -70,3 +70,7 @@ func (s *store) Delete(id string) (err error) {
 	_, err = conn.Do("DEL", key)
 	return
 }
+
+func toMilliseconds(d time.Duration) int64 {
+	return d.Nanoseconds() / 1000000
```

Comment:
Should we follow the implementation in stdlib? https://golang.org/src/time/time.go?s=25703:25738#L793

Reply:
I'll change it to `return int64(d / time.Millisecond)`. Fractional part would be ignored, since Redis can't handle it and the precision is enough.